package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-models-table",
-  "version": "1.11.0-beta.2",
+  "version": "100.0.1",
```

Comment:
prefixing our releases with 100 to allow parallel development with the base project.